README.md

```diff
@@ -0,0 +1,70 @@
+# [<img src="https://ftp.cubrid.org/CUBRID_Docs/Brochure/BI_Images/cubrid-logo-tangram-300.png" width="25" height="25"/>](cubrid_logo.png) CUBRID Database Management System
+
+CUBRID is a DBMS being supported by an active community of open source developers 
```

Comment:
"CUBRID is a DBMS supported by an active..." without the word "being."

Reply:
Thank you
```suggestion
CUBRID is a DBMS supported by an active community of open source developers 
```